database/repos/chained.js

```diff
@@ -0,0 +1,37 @@
+const tables = {
+  users: require('../queries').users,
+  rooms: require('../queries').rooms
+};
+
+
+module.exports = (repo, pgp) => ({
+  initializeDB: () => (
+      repo.tx((t) => {
```

Comment:
I would like some explanation on what both repo and tx is, what it's doing, and where it's coming from.